managedRepository/README.md

```diff
@@ -32,13 +32,14 @@ has_projects
 allow_merge_commit
 allow_rebase_merge
 allow_squash_merge
+auto_init
 topics
 ```
 ## Example
 
 ```
 module "MyReop" {
-  source                      = "./managedRepository"
+  source                      = "github.com/enthought/terraform-modules.git//managedRepository"
```

Comment:
@dbolack can you also rename the directory? `managedRepository` -> `managed_repository`

Reply:
Done